common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -714,6 +714,36 @@ def go_to_tab_and_assert_problem(self, position, problem_name):
         )
         self.assertEqual(self.problem_page.problem_name, problem_name)
 
+    def test_problem_meta_after_submit_and_navigate(self):
+        """
+        Scenario:
+        I go to sequential position 1
+        Facing problem1, I select 'choice_1'
+        Then I click submit button
+        Then I go to sequential position 2
+        Then I came back to sequential position 1 again
+        Facing problem1, I observe the problem1 meta and score data is not
+        altered before and after sequence navigation
+        """
+        # Go to sequential position 1 and assert that we are on problem 1.
+        self.go_to_tab_and_assert_problem(1, self.problem1_name)
+
+        # Update problem 1's content state by clicking check button.
+        self.problem_page.click_choice('choice_choice_1')
+        self.problem_page.click_submit()
+        self.problem_page.wait_for_expected_status('label.choicegroup_incorrect', 'incorrect')
+
+        before_meta = self.problem_page.problem_meta
+
+        # Go to sequential position 2 and assert that we are on problem 2.
+        self.go_to_tab_and_assert_problem(2, self.problem2_name)
+
+        # Come back to our original unit in the sequence and assert that the content hasn't changed.
+        self.go_to_tab_and_assert_problem(1, self.problem1_name)
+
+        after_meta = self.problem_page.problem_meta
+        self.assertEqual(before_meta, after_meta)
```

Comment:
I went back and forth about both of these approaches and decided on a separate focused test for the meta.  I am open to moving it if there is a consensus on that being a better approach.